datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -115,8 +104,12 @@ export class DatasetEntity implements Entity<Dataset> {
             urn: entity.urn,
             name: entity.name,
             type: EntityType.Dataset,
-            upstreamChildren: getChildren(entity, Direction.Upstream).map((child) => child.dataset.urn),
-            downstreamChildren: getChildren(entity, Direction.Downstream).map((child) => child.dataset.urn),
+            upstreamChildren: getChildren({ entity, type: EntityType.Dataset }, Direction.Upstream).map(
+                (child) => child.entity.urn,
+            ),
+            downstreamChildren: getChildren({ entity, type: EntityType.Dataset }, Direction.Downstream).map(
```

Comment:
Can downstream children be entity types other than Dataset? Is this limiting that to just Datasets?

Reply:
this is saying we are getting the children of a dataset urn- it is not specifying the return type 👍